net-irc/quasselgrep/quasselgrep-0_p20200830.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+PYTHON_REQ_USE=( sqlite )
+
+inherit distutils-r1 eutils python-r1
+
+DESCRIPTION="Tool for searching quassel logs from the commandline"
+HOMEPAGE="https://github.com/fish-face/quasselgrep"
+
+MY_COMMIT="9b6b0bc1252daa6e574363d87d04eebd981215a5"
+SRC_URI="https://github.com/fish-face/${PN}/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
No PN in SRC_URI please
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Plaintexting_.24.7BPN.7D

Reply:
Eh, I knew that, somehow I missed it while I was working on this... I updated it. It's always good to have a pair of fresh eyes take a look et the ebuild ;)
Thanks!